public/scripts/sysprompt.js

```diff
@@ -108,7 +105,7 @@ function toggleSystemPromptCallback(_args, state) {
     }
 
     const newState = isTrueBoolean(state);
-    newState ? enableSystemPromptCallback() : disableSystemPromptCallback();
+    newState ? setSystemPromptStateCallback(true) : setSystemPromptStateCallback(false);
```

Comment:
Is this cute lil' thing Copilots doing? :D
```suggestion
    setSystemPromptStateCallback(newState);
```

Reply:
Why did I do this even.

<img src="https://github.com/user-attachments/assets/c9370324-52e9-4005-b8e3-b64a630388a4" width=150>

Reply:
Brainless code refactor/replace, without thinking. I know that feel.